src/index.js

```diff
@@ -45,9 +43,11 @@ export default function InfiniteLoading({
               ref={ref}
               width={width}
             >
-              {({ index, style }) =>
-                <div style={style}>{children({ item: items[index], index })}</div>
-              }
+              {({ index, style }) => (
+                <div style={{ ...style, backgroundColor: '#ff0' }}>
```

Comment:
Can you remove this background color ? You can add it in the example if you want 😄

Reply:
Sure, i forgot to remove it. :stuck_out_tongue_closed_eyes: